experiments/datasetsizes/setup

```diff
@@ -1,11 +1,34 @@
 #!/bin/bash
-# This file sets the experiment up. In case of failures, return a non-ze
-# exit code
 
-set -e
+source environment.conf
 
-#TODO download tiff generator
-#TODO build tiff generator
-#TODO generate input
-#TODO move input to the proper place
-#TODO code issues ..
+rm -rf $results_path
+
+mkdir $code_path
+mkdir $results_path
+
+if [ ! -d $code_path/$promethee_path ]; then
+	git clone $promethee_url -b $promethee_branch
+	mv $promethee_path $code_path
+	git --git-dir=$code_path/$promethee_path/.git checkout --detach $promethee_version
+fi
+
+if [ ! -d $code_path/$tiff_utils_path ]; then
+	git clone $tiff_utils_url
+	mv $tiff_utils_path $code_path
+	git --git-dir=$code_path/$tiff_utils_path/.git checkout --detach $tiff_utils_version
+fi
+
+for (( i=0; i<${#test_cases[@]}; i++ ));
+do
+	if [ ! -e $code_path/$umbu_caja_path/${test_cases[i]}.tif ]; then
+		g++ -std=c++14 $code_path/$tiff_utils_path/tifGenerator.cpp -o $code_path/$tiff_utils_path/tifGenerator -ltiff
+		./$code_path/$tiff_utils_path/tifGenerator ${tiff_sizes[i]} ${test_cases[i]}.tif -R
```

Comment:
at first glance, $code_path reads better than ./$code_path (if you agree change every code line using ./$